### PR TITLE
config: added jrd data load and api yaml files under prod

### PR DESCRIPTION
**Pull request by shashank-rastogi**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: k8s/prod/common/rd/judicial-api.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: rd-judicial-api
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: rd-judicial-api
+    version: 0.0.4
```
Comment (timja) [inline k8s/prod/common/rd/judicial-api.yaml:18]: your base chart  version needs updating before going to prod:
https://github.com/hmcts/rd-judicial-api/blob/master/charts/rd-judicial-api/requirements.yaml#L3

see here for the latest: https://github.com/hmcts/chart-java/releases

File: k8s/prod/common/rd/judicial-data-load.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: rd-judicial-data-load
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: rd-judicial-data-load
+    version: 0.1.1
```
Comment (timja) [inline k8s/prod/common/rd/judicial-data-load.yaml:18]: your base chart  version needs updating before going to prod:
https://github.com/hmcts/rd-judicial-data-load/blob/master/charts/rd-judicial-data-load/requirements.yaml#L3

see here for the latest: https://github.com/hmcts/chart-java/releases

File: k8s/prod/common/rd/judicial-api.yaml (hunk 1 above)
Comment (shashank-rastogi) [reply k8s/prod/common/rd/judicial-api.yaml:18]: okay..will do it now and release the chart version update first1 👍

File: k8s/prod/common/rd/judicial-data-load.yaml (hunk 2 above)
Comment (shashank-rastogi) [reply k8s/prod/common/rd/judicial-data-load.yaml:18]: okay..will do it now and release the chart version update first1 👍

**Pull request merged**